.eslintrc.json

```
@@ -0,0 +1,16 @@
+{
+  "env": {
+    "commonjs": true,
+    "es6": true,
+    "node": true
```

This should fix the errors regarding describe/it not being defined.

```suggestion
    "jest": true,
    "node": true
```